### PR TITLE
Meta: Document fuzz dump

### PR DESCRIPTION
**Pull request by BenWiederhake**
Follow-up to #4242

Hey @devsh0, what do you think about moving the jpg fuzz testcases there? :)

Hey @everybodyelse, what do you think about this? ^^

### CODE REVIEW

**Comment by Lubrsi** [comment]
LGTM

File: Meta/Lagom/ReadMe.md [hunk 1]
```diff
@@ -34,6 +34,24 @@ To run several fuzz jobs in parallel, pass `-jobs=24 -workers=24`.
 To get less log output, pass `-close_fd_mask=3` -- but that but hides assertion messages. Just `1` only closes stdout.
 It's good to move overzealous log output behind `FOO_DEBUG` macros.
 
+### Keeping track of interesting testcases
+
+There are many quirky files that exercise a lot of interesting edge cases.
+We should probably keep track of them, somewhere.
+
+We have a [bmp suite and a jpg suite and several others](https://github.com/SerenityOS/serenity/tree/master/Base/res/html/misc).
+They are are GPL'ed, and therefore not quite as compatible with the rest of Serenity.
```
Comment (Lubrsi) [inline Meta/Lagom/ReadMe.md:43]:
```suggestion
They are GPL'ed, and therefore not quite as compatible with the rest of Serenity.
```

**Comment by devsh0** [comment]
Done!
See #4265 and [this](https://github.com/SerenityOS/serenity-fuzz-corpora/pull/1) PR in the other repo.

**Comment by BenWiederhake** [comment]
Changes since last push:
- Fix embarrassing typo
- Rebase for convenience

**Pull request merged**